pkg/backend/filestate/backend.go

```diff
@@ -323,7 +323,7 @@ func (b *localBackend) GetStack(ctx context.Context, stackRef backend.StackRefer
 	snapshot, path, err := b.getStack(stackName)
 
 	switch {
-	case gcerrors.Code(drillError(err)) == gcerrors.NotFound:
+	case gcerrors.Code(err) == gcerrors.NotFound:
```

Comment:
The problem seems to be that we are passing the unwrapped error to go-cloud's `Code` func which tries to find a matching error that it can assign as `gcerr.Error`. So when it is passed an already-unwrapped error, it fails return the error code that we expect to compare against.

Reply:
Ah, `gcerrors` itself is wrapping errors, so when we unwrap the chain, it prevents `gcerrors.Code` from finding its wrapped error. This is my bad, I had suggested unwrapping the full chain to @iwahbe since we are also wrapping, not realizing that `gcerrors` itself is wrapping and relying on it. Sorry about that @iwahbe.